docs/tutorials/end_to_end_example.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
<p>Do we think it's worthwhile for me to edit the figure caption with a better description of how to interpret the simulation data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Yeah, that would be great!</p>